lib/ex_subtil_backend/workflow/step/generate_dash.ex

```diff
@@ -140,19 +159,20 @@ defmodule ExSubtilBackend.Workflow.Step.GenerateDash do
     build_gpac_parameters(params, result)
   end
 
-  defp convert_gpac_key("segment_duration"), do: "-dash"
-  defp convert_gpac_key("fragment_duration"), do: "-frag"
+  defp convert_gpac_key("segment_duration"), do: :"-dash"
+  defp convert_gpac_key("fragment_duration"), do: :"-frag"
```

Comment:
What is the purpose of returning atoms instead of strings here?

Reply:
atoms are used to define maps like:
```elixir
%{
  key: "value"
}
```

strings defines maps with this syntax:
```elixir
%{
  "key" => "value"
}
```

So I prefers syntax using atoms.

Reply:
OK then!